app/globals.css

```
@@ -295,6 +295,49 @@
+
+/* Scroll-in reveal driven by <Reveal> (components/ui/reveal.tsx) — the CSS
+   replacement for framer-motion's whileInView fade+slide. (#932) */
+[data-reveal="hidden"] {
+  opacity: 0;
+}
+
+[data-reveal="visible"] {
+  animation: revealUp 0.5s ease-out forwards;
+}
+
+@keyframes revealUp {
```

Stylelint (17.13.0)

[error] Expected keyframe name "revealUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)